src/modules/sensors/als.c

```diff
@@ -146,8 +147,12 @@ static int capture(void *dev, double *pct, const int num_captures, char *setting
             }
         }
 
-        if (illuminance >= 0) {
-            pct[ctr++] = illuminance / max;
+        if (illuminance >= 1) {
+            if (illuminance != min) {
+                pct[ctr++] = log10(illuminance) / log10(max);
```

Comment:
Sorry to come back to this, but perhaps
```suggestion
                pct[ctr++] = (log10(illuminance) - log10(min)) / (log10(max) - log10(min));
```
would be best, to use the full scale defined within min and max. That would then require min != max.

Reply:
I think that is ok; the question is: what do we gain doing this way?
I mean, for example:
```
min -> 100
max -> 100000
curr -> 1000


yourProposal: (3 - 2) / (5 -2) -> 0.33
currentWay: 3 / 5 -> 0.6
```
From the doc (https://docs.microsoft.com/en-us/windows/win32/sensorsapi/understanding-and-interpreting-lux-values) it seems like lux values between 1000 and 5000 should be around 0.6 (lighting step 6).

I mean, i am worried we are "stretching" the scale too much with your new proposal.

Reply:
Note, moreover, that min and max were customizable because i didn't really know that there was a kinda logarithmic correlation for lux values.
Now, thanks to your work, i think it is fine to drop those settings, leaving just `interval` as a customizable setting.